src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1411,6 +1441,12 @@ def enable_features(cmd, client, resource_group_name, cluster_name, features, ku
 
     release_namespace = validate_release_namespace(client, cluster_name, resource_group_name, kube_config, kube_context, helm_client_location)
 
+    helm_values = get_all_helm_values(release_namespace, kube_config, kube_context, helm_client_location)
+
+    if helm_values.get('global').get('isLeastPrivilegesMode') is True:
```

Comment:
have you added this check in disable features?

Reply:
To disable a feature user shouldn't need any role update and hence this warning is not necessary for disable features. In case of enable , a feature might need higher access and hence the warning